apps/designer/app/designer/features/sidebar-left/panels/image-upload.tsx

```diff
@@ -0,0 +1,58 @@
+import { ImageIcon } from "~/shared/icons";
+import { Button, Flex, Grid, Heading } from "~/shared/design-system";
+import { useRef } from "react";
+import { Form } from "@remix-run/react";
+import { Asset } from "@prisma/client";
+import { Image } from "~/shared/design-system/components/image";
+
+export const TabContent = ({ assets }: { assets: Array<Asset> }) => {
+  const inputRef = useRef<HTMLInputElement | null>(null);
+  const submitButtonRef = useRef<HTMLButtonElement | null>(null);
+
+  const onChange = () => {
+    if (submitButtonRef.current) {
+      submitButtonRef.current.click();
+    }
+  };
+
+  return (
+    <Flex gap="3" direction="column" css={{ padding: "$1" }}>
+      <Flex
+        css={{
+          justifyContent: "space-between",
+        }}
+      >
+        <Heading>Assets</Heading>
+        <Form method="post" encType="multipart/form-data">
+          <input
+            accept="image/*"
+            type="file"
+            name="image"
+            ref={inputRef}
+            onChange={onChange}
+            style={{ display: "none" }}
+          />
+          <button
```

Comment:
hmm why do you need 2 buttons?

Reply:
ah got it, Button is for selecting files, button is for initiating submit event

Reply:
yap!